crates/unftp-auth-pam/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "unftp-auth-pam"
+version = "0.1.0"
+authors = [
+    "Agoston Horvath <ahorvath@bol.com>",
+    "Dávid Kosztka <dkosztka@bol.com>",
+    "Hannes de Jager <hdejager@bol.com>",
+    "Koen Wilde <koen@chillheid.nl>",
+    "Rob klein Gunnewiek <rkleingunnewiek@bol.com>",
+]
+edition = "2018"
+license = "Apache-2.0"
+keywords = ["libunftp", "unftp", "ftp", "ftps", "pam", "auth", "authenticator"]
+categories = ["network-programming"]
+documentation = "https://docs.rs/unftp-auth-pam"
+homepage = "https://github.com/bolcom/libunftp/tree/master/crates/unftp-auth-pam"
+repository = "https://github.com/bolcom/libunftp/tree/master/crates/unftp-auth-pam"
+readme = "README.md"
+
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-trait = "0.1.42"
+libunftp = { version="0.16.2", path="../../"}
+tracing = "0.1.23"
+tracing-attributes = "0.1.12"
+
+[target.'cfg(unix)'.dependencies]
```

Comment:
question(non-blocking): now that it is not part of libunftp, is this still the most idiomatic way to prevent this to be built on systems without `pam`?

Reply:
I think this line needs to be removed.